plenum/test/waits.py

```diff
@@ -107,7 +107,7 @@ def expectedClientRequestPropagationTime(nodeCount):
 
 
 def expectedTransactionExecutionTime(nodeCount):
-    return int(CLIENT_REPLY_TIMEOUT * nodeCount)
+    return int(2 * CLIENT_REPLY_TIMEOUT * nodeCount)
```

Comment:
Why we are doubling the expected time? And won't this impact any test which ultimately depends on this function?